cmd/admin-handlers_test.go

```diff
@@ -162,3 +163,151 @@ func TestServiceStopHandler(t *testing.T) {
 func TestServiceRestartHandler(t *testing.T) {
 	testServicesCmdHandler(restartCmd, t)
 }
+
```

Comment:
Yes, I will add comments in the next update. Awaiting more reviews before updating the PR.